tests/test_group.rs

```diff
@@ -302,18 +311,15 @@ fn group_operations() {
         };
 
         // Make sure that both groups have the same public tree
-        if group_alice_1234.tree().public_key_tree() != group_alice_1234.tree().public_key_tree() {
-            _print_tree(&group_alice_1234.tree(), "Alice added Bob");
+        if group_alice.tree().public_key_tree() != group_bob.tree().public_key_tree() {
+            _print_tree(&group_alice.tree(), "Alice added Bob");
             panic!("Different public trees");
         }
 
-        // Alice sends a message to Bob
+        // === Alice sends a message to Bob ===
         let message_alice = [1, 2, 3];
-        let mls_ciphertext_alice = group_alice_1234.create_application_message(
-            &[],
-            &message_alice,
-            &alice_credential_bundle,
-        );
+        let mls_ciphertext_alice =
+            group_alice.create_application_message(&[], &message_alice, &alice_credential_bundle);
         let mls_plaintext_bob = match group_bob.decrypt(mls_ciphertext_alice) {
```

Comment:
Maybe test that alice's message is equal to the message decrypted by Bob?

Reply:
That's what we are testing in the lines below.

Reply:
Ah, sorry, I didn't expand the lines hidden by GH. Right you are!